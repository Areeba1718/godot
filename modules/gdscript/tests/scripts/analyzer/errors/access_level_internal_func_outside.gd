@private

class Foo:
    internal func foo():
        pass

func test():
    Foo.new().foo()
