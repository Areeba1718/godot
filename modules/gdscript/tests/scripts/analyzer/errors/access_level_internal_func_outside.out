GDTEST_ANALYZER_ERROR
Cannot call internal function "foo()" on the class Foo.
